spec/jobs/scraping_process_distributing_job_spec.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe ScrapingProcessDistributingJob, type: :job do
+  include ActiveJob::TestHelper
```

Comment:
can we go without this?

Reply:
No we cannot because I used method from that module which is `assert_enqueued_with`